### PR TITLE
update default k8s version to v1.30

### PR DESCRIPTION
**Pull request by tapas4java**
Updating default k8s version

### CODE REVIEW

**Review by nabuskey — APPROVED** [review]
LGTM

**Comment by OneCricketeer** [comment]
Should `ENVTEST_K8S_VERSION` be updated as well?

File: pkg/cmd/create/root.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func init() {
 	// cluster related flags
 	CreateCmd.PersistentFlags().BoolVar(&recreateCluster, "recreate", false, "Delete cluster first if it already exists.")
 	CreateCmd.PersistentFlags().StringVar(&buildName, "build-name", "localdev", "Name for build (Prefix for kind cluster name, pod names, etc).")
-	CreateCmd.PersistentFlags().StringVar(&kubeVersion, "kube-version", "v1.29.2", "Version of the kind kubernetes cluster to create.")
+	CreateCmd.PersistentFlags().StringVar(&kubeVersion, "kube-version", "v1.30.0", "Version of the kind kubernetes cluster to create.")
```
Comment (OneCricketeer) [inline pkg/cmd/create/root.go:50]: Why not latest patch release? 1.30.4?

**Comment by cmoulliard** [comment]
A table about the kubernetes version(s) supported should be included part of the documentation and release note !

File: pkg/cmd/create/root.go (hunk 1 above)
Comment (nabuskey) [reply pkg/cmd/create/root.go:50]: Yeah good idea. I looked at the kind version we are using and its default k8s version (`v1.30.0`).  Bumping it up to the next patch release is a good idea. Your PR would be awesome here 😃

**Comment by nabuskey** [comment]
Agreed. I really would love to automate this bit for sure.

**Pull request merged**